sample-apps/python-manual-instrumentation-sample-app/app.py

```diff
@@ -63,9 +64,10 @@ def setup_opentelemetry(tracer, meter):
     # Set up AWS X-Ray Propagator
     propagate.set_global_textmap(AwsXRayPropagator())
     # Service name is required for most backends
-    resource = Resource(attributes={
-        SERVICE_NAME: 'python-manual-instrumentation-sampleapp'
-    })
+    resource_attributes = { 'service.name': 'python-manual-instrumentation-sample-app' }
+    if (os.environ.get("OTEL_RESOURCE_ATTRIBUTES")):
+        resource_attributes = None
```

Comment:
I specify resource attributes as none here if it is specified in the environment because in the Python SDK, if no resource attributes are provided, then it will use the attributes from the environment in `OTEL_RESOURCE_ATTRIBUTES` by default which is what we would want.  However, if attributes in the environment do not exist, then I provide resource attributes as specified in line 67 (usually for local testing).